graylog2-server/src/main/java/org/graylog2/contentpacks/facades/StreamFacade.java

```diff
@@ -268,8 +269,12 @@ public void delete(Stream nativeEntity) {
             return;
         }
         try {
+            LOG.debug("WTF");
```

Comment:
Is this a debugging leftover or should we add a real debug message?

Reply:
@bernd  sorry about that I forgot to push, my bad